polartools/diffraction.py

```diff
@@ -46,7 +46,7 @@
 )
 
 
-def fit_peak(xdata, ydata, model="Gaussian", scan=None, output=False):
+def fit_peak(xdata, ydata, model="Gaussian"):
```

Comment:
define the models using an enumeration (helps to write the code more clearly):

```python
In [11]: import lmfit.models

In [12]: class Model(Enum):
    ...:     Gaussian = lmfit.models.GaussianModel
    ...:     Lorentzian = lmfit.models.LorentzianModel
    ...:     PseudoVoigt = lmfit.models.LorentzianModel
    ...: 

In [13]: Model.Gaussian
Out[13]: <Model.Gaussian: <class 'lmfit.models.GaussianModel'>>

```

so the kwarg would have the default: `model=Model.Gaussian`

Reply:
then, call lmfit with this: `model.value()`